frontend/src/components/Forms/ReadForm/ReadField/index.tsx

```diff
@@ -100,9 +100,14 @@ export const ReadField = ({
               <ListSubheader sx={{ bgcolor: "#2d2b2b", color: "#fff" }}>{title}</ListSubheader>
             }
           >
-            {(value as unknown as { link: string; label: string }[]).map((value) => {
+            {(
+              value as unknown as {
+                link: string;
+                label: string;
+              }[]
+            ).map((value) => {
               return (
-                <>
+                <div key={value?.link /* prevent key warning in console */}>
```

Comment:
I think if there are ever two `value`s with the same `link` this will still throw an error. You could change line 108 to provide the index of the loop (`map(value, index)` and use that as the key which should always be unique.

Reply:
good idea. I"ll do that if the error comes back again